src/Microsoft.PowerShell.ConsoleHost/host/msh/ProgressNode.cs

```diff
@@ -432,7 +435,7 @@ internal static bool IsMinimalProgressRenderingEnabled()
                     barLength = barWidth - 1;
                 }
 
-                if (barLength < sb.Length)
+                if (barLength < rawUI.LengthInBufferCells(sb.ToString()))
```

Comment:
Same here, `rawUI.LengthInBufferCells(sb.ToString())` will return the incorrect value as it counts the `Reverse` VT sequences in. It would be the best if you can insert the `Reverse` prefix when adding the `ReverseOff` sequences.

Reply:
This one doesn't seem to be resolved.